internal/commands/event_invoker.go

```diff
@@ -42,12 +42,16 @@ func newEventInvoker(app *app.App) *eventInvoker {
 }
 
 func (e *eventInvoker) eventInvoker(ctx context.Context, cmdLine string, event TriggerEvent) error {
+	logWriter := e.app.Logger.With(
```

Comment:
The `event` is already available at this point, so it can also be embedded in the `logWriter`.